diff --git a/apps/standard/lcm_relay.py b/apps/advanced/ui.py
index c4bbc3fb..0586a84d 100755
--- a/apps/standard/lcm_relay.py
+++ b/apps/advanced/ui.py
@@ -1,5 +1,7 @@
 #!/usr/bin/env python3
 
+from importlib.resources import files
+
 import numpy as np
 
 # API imports
@@ -7,7 +9,9 @@ from pntos.api import EstimateWithCovariance, EstimateWithCovarianceType, Loggin
 
 # Import Cobra plugins and config structs
 from pntos.cobra import (
+    CobraUiLogPlayerPlugin,
     EkfFusionStrategyPlugin,
+    ExperimentalCobraUiPlugin,
     LcmTransportPlugin,
     ManualHeadingAlignInitializationPlugin,
     StandardControllerPlugin,
@@ -21,6 +25,7 @@ from pntos.cobra import (
 )
 from pntos.cobra.config import (
     ControllerConfig,
+    ExperimentalCobraUiConfig,
     FogmConfig,
     FogmStateBlockConfig,
     FusionEngineConfig,
@@ -36,6 +41,12 @@ from pntos.cobra.config import (
     TimeBiasConfig,
 )
 
+static_assets_path: str = (
+    files('pntos.cobra')  # type: ignore[call-arg, attr-defined]
+    .joinpath('advanced_plugins', 'ui', '_static', 'dist')
+    .as_posix()
+)
+
 # Config setup
 C_imu_to_platform = (
     (0.99802515, 0.01772605, 0.06026269),
@@ -123,6 +134,9 @@ my_config = [
             ),
         ),
     ),
+    ExperimentalCobraUiConfig(
+        group='config/cobra_ui', static_folder=static_assets_path
+    ),
 ]
 # End Config
 
@@ -144,6 +158,8 @@ plugins = [
     StandardRegistryPlugin('Cobra Standard Registry Plugin', config=my_config),
     StandardPreprocessorPlugin('Cobra Standard Preprocessor Plugin'),
     StandardOrchestrationPlugin('Cobra Standard Orchestration Plugin'),
+    ExperimentalCobraUiPlugin('Experimental Cobra UI Plugin'),
+    CobraUiLogPlayerPlugin('Cobra UI Log Player Plugin'),
 ]
 
 # Start the controller, and pass it all of the other plugins to use
