diff --git a/apps/standard/pos_vel_ins.py b/apps/standard/posvel_ins.py
index c583d519..fa004152 100755
--- a/apps/standard/pos_vel_ins.py
+++ b/apps/standard/posvel_ins.py
@@ -1,11 +1,13 @@
 #!/usr/bin/env python3
 
-"""App to process IMU, position, and velocity measurements.
-
+"""App to process IMU, position and velocity measurements.
 
 Note: This app uses a simpler position measurement model than some of the other apps.
 Thus, the filter solution is less accurate than those apps.
-"""
+
+Note: This is the same data as the pos_vel_ins tutorial app, but with the position
+and NED velocity measurements combined into a PVA measurement to be processed as a
+single measurement."""
 
 import sys
 
@@ -38,9 +40,8 @@ from pntos.cobra.config import (
     InertialConfig,
     LcmLogTransportConfig,
     ManualHeadingAlignmentConfig,
-    PinsonPositionMPConfig,
     PinsonStateBlockConfig,
-    PinsonVelocityMPConfig,
+    PosVelMPConfig,
     StandardOrchestrationConfig,
     TimeAdjusterConfig,
     TimeBiasConfig,
@@ -73,7 +74,7 @@ my_config = [
         channels_to_process=(
             '/sensor/vn-100/imu',
             '/sensor/ublox-ZED-F9T/position',
-            '/sensor/ublox-ZED-F9T/velocity',
+            '/sensor/ublox-ZED-F9T/pva',
         ),
     ),
     ControllerConfig(),
@@ -104,19 +105,13 @@ my_config = [
             ),
         ),
         mp_configs=(
-            PinsonPositionMPConfig(
-                group='config/pos_measurement_processor',
-                label='pos',
-                channel='/sensor/ublox-ZED-F9T/position',
+            PosVelMPConfig(
+                group='config/posvel_measurement_processor',
+                label='posvel',
+                channel='/sensor/ublox-ZED-F9T/pva',
                 state_block_labels=('pinson15',),
                 lever_arm=(-0.50, 0.38, -0.05),
             ),
-            PinsonVelocityMPConfig(
-                group='config/vel_measurement_processor',
-                label='vel',
-                channel='/sensor/ublox-ZED-F9T/velocity',
-                state_block_labels=('pinson15',),
-            ),
         ),
         inertial_config=InertialConfig(
             group='config/inertial',
@@ -145,10 +140,7 @@ my_config = [
             ),
             TimeBiasConfig(
                 group='config/time_bias',
-                channels_to_correct=(
-                    '/sensor/ublox-ZED-F9T/position',
-                    '/sensor/ublox-ZED-F9T/velocity',
-                ),
+                channels_to_correct=('/sensor/ublox-ZED-F9T/pva',),
                 time_bias=int(0.15 * 1e9),
             ),
         ),
