diff --git a/apps/standard/pos_ins.py b/apps/standard/pos_ins_leverarm.py
index 86eea2f3..f61edb85 100755
--- a/apps/standard/pos_ins.py
+++ b/apps/standard/pos_ins_leverarm.py
@@ -33,7 +33,7 @@ from pntos.cobra.config import (
     LcmLogTransportConfig,
     ManualHeadingAlignmentConfig,
     PinsonStateBlockConfig,
-    PinsonWithNedFogmPositionMPConfig,
+    PinsonWithLeverArmPositionMPConfig,
     StandardOrchestrationConfig,
     TimeAdjusterConfig,
     TimeBiasConfig,
@@ -95,14 +95,33 @@ my_config = [
                     tau=(300.0, 300.0, 200.0),
                 ),
             ),
+            FogmStateBlockConfig(
+                group='config/lever_arm_fogm_block',
+                label='pos_sensor_lever_arm',
+                estimate_with_covariance=EstimateWithCovariance(
+                    type=EstimateWithCovarianceType.EWC_GENERIC,
+                    estimate=np.zeros((3,)),
+                    covariance=(np.eye(3) * 100.0),
+                ),
+                fogm_model=FogmConfig(
+                    group='config/pos_sensor_lever_arm',
+                    sigma=(2.0, 2.0, 2.0),
+                    tau=(1e5, 1e5, 1e5),
+                ),
+            ),
         ),
         mp_configs=(
-            PinsonWithNedFogmPositionMPConfig(
+            PinsonWithLeverArmPositionMPConfig(
                 group='config/pos_measurement_processor',
                 label='pos',
                 channel='/sensor/ublox-ZED-F9T/position',
-                state_block_labels=('pinson15', 'pos_sensor_error'),
-                lever_arm=(-0.50, 0.38, -0.05),
+                state_block_labels=(
+                    'pinson15',
+                    'pos_sensor_error',
+                    'pos_sensor_lever_arm',
+                ),
+                # Initial lever arm incorrect to demonstrate estimation
+                lever_arm=(-8.50, 8.38, -8.05),
             ),
         ),
         inertial_config=InertialConfig(
