diff --git a/apps/standard/pos_ins.py b/apps/standard/pos_ins_leverarm.py
index d40a1834..290914e4 100755
--- a/apps/standard/pos_ins.py
+++ b/apps/standard/pos_ins_leverarm.py
@@ -96,18 +96,40 @@ my_config = [
                     tau=(300.0, 300.0, 200.0),
                 ),
             ),
+            FogmStateBlockConfig(
+                group='config/lever_arm_fogm_block',
+                label='pos_sensor_lever_arm',
+                estimate_with_covariance=EstimateWithCovariance(
+                    type=EstimateWithCovarianceType.EWC_GENERIC,
+                    estimate=np.zeros((3,)),
+                    covariance=(np.eye(3) * 100.0),
+                ),
+                fogm_model=FogmConfig(
+                    group='config/pos_sensor_lever_arm',
+                    sigma=(2.0, 2.0, 2.0),
+                    tau=(1e5, 1e5, 1e5),
+                ),
+            ),
         ),
         mp_configs=(
             SensorMeasurementProcessorConfig(
                 group='config/pos_measurement_processor',
-                identifier='pinson_with_ned_fogm_position',
+                identifier='pinson_with_lever_arm_position',
                 label='pos',
                 channel='/sensor/ublox-ZED-F9T/position',
-                state_block_labels=('pinson15', 'pos_sensor_error'),
+                state_block_labels=(
+                    'pinson15',
+                    'pos_sensor_error',
+                    'pos_sensor_lever_arm',
+                ),
                 aux_channels=('INERTIAL_PVA',),
                 sensor_config=SensorConfig(
                     group='config/gp3d_state_modeling',
-                    lever_arm=(-0.50, 0.38, -0.05),
+                    lever_arm=(
+                        -8.50,
+                        8.38,
+                        -8.05,
+                    ),  # Initial lever arm incorrect to demonstrate estimation
                     orientation=(0.0, 0.0, 0.0, 0.0),
                     sensor_name='position',
                 ),
