diff --git a/apps/tutorial/pos_ins.py b/apps/tutorial/pos_vel_ins.py
index ba251865..0b5b34cd 100755
--- a/apps/tutorial/pos_ins.py
+++ b/apps/tutorial/pos_vel_ins.py
@@ -17,7 +17,7 @@ from pntos.cobra import (
     StandardRegistryPlugin,
     TutorialInitializationPlugin,
     TutorialPosInsStateModelingPlugin,
-    TutorialPosOrchestrationPlugin,
+    TutorialPosVelOrchestrationPlugin,
     UiLogPlottingPlugin,
 )
 from pntos.cobra.config import (
@@ -102,6 +102,7 @@ my_config = [
     TutorialOrchestrationConfig(
         position_channel='/sensor/ublox-ZED-F9T/position',
         group='config/orchestration',
+        velocity_channel='/sensor/ublox-ZED-F9T/velocity',
     ),
     TimeAdjusterConfig(
         group='config/time_adjuster',
@@ -121,7 +122,10 @@ my_config = [
     ),
     TimeBiasConfig(
         group='config/time_bias',
-        channels_to_correct=('/sensor/ublox-ZED-F9T/position',),
+        channels_to_correct=(
+            '/sensor/ublox-ZED-F9T/position',
+            '/sensor/ublox-ZED-F9T/velocity',
+        ),
         time_bias=int(0.15 * 1e9),
     ),
 ]
@@ -143,7 +147,7 @@ plugins = [
     StandardRegistryPlugin('Cobra Standard Registry Plugin', config=my_config),
     StandardPreprocessorPlugin('Cobra Standard Preprocessor Plugin'),
     UiLogPlottingPlugin('Cobra UI Logfile Plotting Plugin'),
-    TutorialPosOrchestrationPlugin('Cobra Tutorial Orchestration Plugin'),
+    TutorialPosVelOrchestrationPlugin('Cobra Tutorial Orchestration Plugin'),
 ]
 
 # Start the controller, and pass it all of the other plugins to use
