diff --git a/pntos-cobra-apps/src/pntos/apps/standard/pos_ins.py b/pntos-cobra-apps/src/pntos/apps/standard/pos_ins_leverarm.py
index 9903fdf3..bf1e191c 100755
--- a/pntos-cobra-apps/src/pntos/apps/standard/pos_ins.py
+++ b/pntos-cobra-apps/src/pntos/apps/standard/pos_ins_leverarm.py
@@ -38,7 +38,7 @@ from pntos.cobra.config import (
     LcmLogTransportConfig,
     ManualHeadingAlignmentConfig,
     PinsonStateBlockConfig,
-    PinsonWithNedFogmPositionMPConfig,
+    PinsonWithLeverArmPositionMPConfig,
     StandardOrchestrationConfig,
     TimeAdjusterConfig,
     TimeBiasConfig,
@@ -47,7 +47,7 @@ from pntos_python_datasets_lcm import EXAMPLE_LCM_LOG
 
 
 def main() -> None:
-    """Run the pos_ins app."""
+    """Run the pos_ins_leverarm app."""
     OUTPUT_LOG = sys.argv[1] if len(sys.argv) > 1 else 'pntos_output.log'
 
     # Config setup
@@ -102,14 +102,33 @@ def main() -> None:
                         tau=(300.0, 300.0, 200.0),
                     ),
                 ),
+                FogmStateBlockConfig(
+                    group='config/lever_arm_fogm_block',
+                    label='pos_sensor_lever_arm',
+                    estimate_with_covariance=EstimateWithCovariance(
+                        type=EstimateWithCovarianceType.EWC_GENERIC,
+                        estimate=np.zeros((3,)),
+                        covariance=(np.eye(3) * 100.0),
+                    ),
+                    fogm_model=FogmConfig(
+                        group='config/pos_sensor_lever_arm',
+                        sigma=(2.0, 2.0, 2.0),
+                        tau=(1e5, 1e5, 1e5),
+                    ),
+                ),
             ),
             mp_configs=(
-                PinsonWithNedFogmPositionMPConfig(
+                PinsonWithLeverArmPositionMPConfig(
                     group='config/pos_measurement_processor',
                     label='pos',
                     channel='/sensor/ublox-ZED-F9T/position',
-                    state_block_labels=('pinson15', 'pos_sensor_error'),
-                    lever_arm=(-0.50, 0.38, -0.05),
+                    state_block_labels=(
+                        'pinson15',
+                        'pos_sensor_error',
+                        'pos_sensor_lever_arm',
+                    ),
+                    # Initial lever arm incorrect to demonstrate estimation
+                    lever_arm=(-8.50, 8.38, -8.05),
                 ),
             ),
             inertial_config=InertialConfig(
