diff --git a/apps/standard/pos_ins.py b/apps/standard/pos_ins_bodyvel.py
index 832536b2..ea24f16d 100755
--- a/apps/standard/pos_ins.py
+++ b/apps/standard/pos_ins_bodyvel.py
@@ -68,6 +68,7 @@ my_config = [
         channels_to_process=(
             '/sensor/vn-100/imu',
             '/sensor/ublox-ZED-F9T/position',
+            '/sensor/simulated/velocity',
         ),
     ),
     ControllerConfig(group='controller'),
@@ -111,6 +112,20 @@ my_config = [
                     sensor_name='position',
                 ),
             ),
+            SensorMeasurementProcessorConfig(
+                group='config/bodyvel_measurement_processor',
+                identifier='pinson_body_velocity',
+                label='body_vel',
+                channel='/sensor/simulated/velocity',
+                state_block_labels=('pinson15',),
+                aux_channels=('INERTIAL_PVA', 'INERTIAL_FORCES_AND_RATES'),
+                sensor_config=SensorConfig(
+                    group='config/bv3d_state_modeling',
+                    lever_arm=(0.0, 0.0, 0.0),
+                    orientation=(0.0, 0.0, 0.0, 0.0),
+                    sensor_name='velocity',
+                ),
+            ),
         ),
         inertial_config=InertialConfig(
             group='config/inertial',
