diff --git a/apps/fusion_gps_ins.py b/apps/fusion_gps_vel_ins.py
index d5b17e83..ca45c837 100755
--- a/apps/fusion_gps_ins.py
+++ b/apps/fusion_gps_vel_ins.py
@@ -9,7 +9,7 @@ from pntos.cobra import (
     LcmTransportPlugin,
     SimpleControllerPlugin,
     SimpleGpsInsStateModelingPlugin,
-    SimpleGpsOrchestrationPlugin,
+    SimpleGpsVelOrchestrationPlugin,
     StandardFusionPlugin,
     StandardInertialPlugin,
     StandardLoggingPlugin,
@@ -85,6 +85,7 @@ my_config = [
     OrchestrationConfig(
         gps_channel='/sensor/ublox-ZED-F9T/position',
         group='config/orchestration',
+        velocity_channel='/sensor/ublox-ZED-F9T/velocity',
     ),
     TimeAdjusterConfig(
         group='config/time_adjuster',
@@ -107,7 +108,7 @@ plugins = [
         'Cobra Standard Logging Plugin',
         global_log_level=LoggingLevel.INFO,  # Switch to `DEBUG` for more informative log output
     ),
-    SimpleGpsOrchestrationPlugin('Cobra Simple Orchestration Plugin'),
+    SimpleGpsVelOrchestrationPlugin('Cobra Simple Orchestration Plugin'),
     StandardRegistryPlugin('Cobra Standard Registry Plugin', config=my_config),
     StandardPreprocessorPlugin('Cobra Standard Preprocessor Plugin'),
 ]
