diff --git a/apps/standard/pos_ins.py b/apps/standard/pos_ins_bodyvel.py
index 8cc11962..915a1b29 100755
--- a/apps/standard/pos_ins.py
+++ b/apps/standard/pos_ins_bodyvel.py
@@ -29,6 +29,7 @@ from pntos.cobra import (
 from pntos.cobra.config import (
     BaseConfig,
     ControllerConfig,
+    DownsamplerConfig,
     FogmConfig,
     FogmStateBlockConfig,
     FusionEngineConfig,
@@ -37,6 +38,7 @@ from pntos.cobra.config import (
     InertialConfig,
     LcmLogTransportConfig,
     ManualHeadingAlignmentConfig,
+    PinsonBodyVelocityMPConfig,
     PinsonStateBlockConfig,
     PinsonWithNedFogmPositionMPConfig,
     StandardOrchestrationConfig,
@@ -71,6 +73,7 @@ my_config: list[BaseConfig] = [
         channels_to_process=(
             '/sensor/vn-100/imu',
             '/sensor/ublox-ZED-F9T/position',
+            '/sensor/simulated/velocity',
         ),
     ),
     ControllerConfig(),
@@ -108,6 +111,14 @@ my_config: list[BaseConfig] = [
                 state_block_labels=('pinson15', 'pos_sensor_error'),
                 lever_arm=(-0.50, 0.38, -0.05),
             ),
+            PinsonBodyVelocityMPConfig(
+                group='config/bodyvel_measurement_processor',
+                label='body_vel',
+                channel='/sensor/simulated/velocity',
+                state_block_labels=('pinson15',),
+                lever_arm=(0.0, 0.0, 0.0),
+                orientation=(1.0, 0.0, 0.0, 0.0),
+            ),
         ),
         inertial_config=InertialConfig(
             group='config/inertial',
@@ -139,6 +150,11 @@ my_config: list[BaseConfig] = [
                 channels=('/sensor/ublox-ZED-F9T/position',),
                 time_bias=int(0.15 * 1e9),
             ),
+            DownsamplerConfig(
+                group='config/downsampler',
+                channels=('/sensor/simulated/velocity',),
+                downsampling_factors=(10,),  # Downsample from 1 Hz to 0.1 Hz
+            ),
         ),
     ),
 ]
