diff --git a/apps/standard/gps_ins.py b/apps/standard/gps_ins_bodyvel.py
index 12295587..a8c127ef 100755
--- a/apps/standard/gps_ins.py
+++ b/apps/standard/gps_ins_bodyvel.py
@@ -106,6 +106,19 @@ my_config = [
                     sensor_name='position',
                 ),
             ),
+            SensorMeasurementProcessorConfig(
+                group='config/bodyvel_measurement_processor',
+                identifier='pinson_body_velocity',
+                label='body_vel',
+                channel='/sensor/simulated/velocity',
+                state_block_labels=('pinson15',),
+                sensor_config=SensorConfig(
+                    group='config/bv3d_state_modeling',
+                    lever_arm=(0.0, 0.0, 0.0),
+                    orientation=(0.0, 0.0, 0.0, 0.0),
+                    sensor_name='velocity',
+                ),
+            ),
         ),
         inertial_config=InertialConfig(
             group='config/inertial',
