diff --git a/apps/standard/pos_ins.py b/apps/standard/pos_vel_ins.py
index 48ea5e79..2a3adef6 100755
--- a/apps/standard/pos_ins.py
+++ b/apps/standard/pos_vel_ins.py
@@ -1,5 +1,12 @@
 #!/usr/bin/env python3
 
+"""App to process IMU, position, and velocity measurements.
+
+
+Note: This app uses a simpler position measurement model than some of the other apps.
+Thus, the filter solution is less accurate than those apps.
+"""
+
 import sys
 
 import numpy as np
@@ -32,8 +39,9 @@ from pntos.cobra.config import (
     InertialConfig,
     LcmLogTransportConfig,
     ManualHeadingAlignmentConfig,
+    PinsonPositionMPConfig,
     PinsonStateBlockConfig,
-    PinsonWithNedFogmPositionMPConfig,
+    PinsonVelocityMPConfig,
     StandardOrchestrationConfig,
     TimeAdjusterConfig,
     TimeBiasConfig,
@@ -67,6 +75,7 @@ my_config = [
         channels_to_process=(
             '/sensor/vn-100/imu',
             '/sensor/ublox-ZED-F9T/position',
+            '/sensor/ublox-ZED-F9T/velocity',
         ),
     ),
     ControllerConfig(),
@@ -97,13 +106,19 @@ my_config = [
             ),
         ),
         mp_configs=(
-            PinsonWithNedFogmPositionMPConfig(
+            PinsonPositionMPConfig(
                 group='config/pos_measurement_processor',
                 label='pos',
                 channel='/sensor/ublox-ZED-F9T/position',
-                state_block_labels=('pinson15', 'pos_sensor_error'),
+                state_block_labels=('pinson15',),
                 lever_arm=(-0.50, 0.38, -0.05),
             ),
+            PinsonVelocityMPConfig(
+                group='config/vel_measurement_processor',
+                label='vel',
+                channel='/sensor/ublox-ZED-F9T/velocity',
+                state_block_labels=('pinson15',),
+            ),
         ),
         inertial_config=InertialConfig(
             group='config/inertial',
@@ -132,7 +147,10 @@ my_config = [
             ),
             TimeBiasConfig(
                 group='config/time_bias',
-                channels_to_correct=('/sensor/ublox-ZED-F9T/position',),
+                channels_to_correct=(
+                    '/sensor/ublox-ZED-F9T/position',
+                    '/sensor/ublox-ZED-F9T/velocity',
+                ),
                 time_bias=int(0.15 * 1e9),
             ),
         ),
