diff --git a/apps/standard/gps_ins.py b/apps/advanced/gps_ins_ros.py
index 190c4b95..a5b9a349 100755
--- a/apps/standard/gps_ins.py
+++ b/apps/advanced/gps_ins_ros.py
@@ -1,7 +1,5 @@
 #!/usr/bin/env python3
 
-import sys
-
 import numpy as np
 
 # API imports
@@ -10,7 +8,6 @@ from pntos.api import EstimateWithCovariance, EstimateWithCovarianceType, Loggin
 # Import Cobra plugins and config structs
 from pntos.cobra import (
     EkfFusionStrategyPlugin,
-    LcmLogTransportPlugin,
     ManualHeadingAlignInitializationPlugin,
     SimpleControllerPlugin,
     StandardFusionPlugin,
@@ -21,14 +18,13 @@ from pntos.cobra import (
     StandardPreprocessorPlugin,
     StandardRegistryPlugin,
 )
+from pntos.cobra.advanced_plugins import Aspn23RosTransportPlugin
 from pntos.cobra.config import (
-    AspnVersion,
     FogmConfig,
     FogmStateBlockConfig,
     ImuConfig,
     ImuRotatorConfig,
     InertialConfig,
-    LcmLogTransportConfig,
     ManualHeadingAlignmentConfig,
     PinsonStateBlockConfig,
     SensorConfig,
@@ -37,9 +33,6 @@ from pntos.cobra.config import (
     TimeAdjusterConfig,
     TimeBiasConfig,
 )
-from pntos_python_datasets import EXAMPLE_LCM_LOG
-
-OUTPUT_LOG = sys.argv[1] if len(sys.argv) > 1 else 'pntos_output.log'
 
 # Config setup
 C_imu_to_platform = (
@@ -59,16 +52,10 @@ imu_model = ImuConfig(
     gyro_bias_initial_sigma=(0.003, 0.003, 0.003),
 )
 my_config = [
-    LcmLogTransportConfig(
-        input_file=EXAMPLE_LCM_LOG,
-        output_file=OUTPUT_LOG,
-        output_version=AspnVersion.V23,
-        group='config/lcm_log_transport',
-    ),
     StandardOrchestrationConfig(
         best_sol_channel='/solution/pntos/pva',
         imu_sol_channel='/solution/pntos-imu/pva',
-        alignment_channels=['/sensor/ublox-ZED-F9T/position', '/sensor/vn-100/imu'],
+        alignment_channels=['/sensor/ublox_ZED_F9T/position', '/sensor/vn_100/imu'],
         pinson_sb_config=PinsonStateBlockConfig(
             group='config/pinson_block',
             identifier='pinson15',
@@ -97,7 +84,7 @@ my_config = [
                 group='config/gps_measurement_processor',
                 identifier='pinson_with_ned_fogm_position',
                 label='gps',
-                channel='/sensor/ublox-ZED-F9T/position',
+                channel='/sensor/ublox_ZED_F9T/position',
                 state_block_labels=['pinson15', 'pos_fogm'],
                 sensor_config=SensorConfig(
                     group='config/gp3d_state_modeling',
@@ -110,7 +97,7 @@ my_config = [
         inertial_config=InertialConfig(
             group='config/inertial',
             expected_dt=0.01,
-            channel='/sensor/vn-100/imu',
+            channel='/sensor/vn_100/imu',
             C_imu_to_platform=C_imu_to_platform,
             inertial_buffer_length=10.0,
         ),
@@ -125,13 +112,13 @@ my_config = [
             ImuRotatorConfig(
                 group='config/imu_rotator',
                 identifier='imu_rotator',
-                channel='/sensor/vn-100/imu',
+                channel='/sensor/vn_100/imu',
                 C_imu_to_platform=C_imu_to_platform,
             ),
             TimeAdjusterConfig(
                 group='config/time_adjuster',
                 identifier='time_adjuster',
-                channel_to_correct='/sensor/vn-100/imu',
+                channel_to_correct='/sensor/vn_100/imu',
                 expected_dt_nsec=int(0.01 * 1e9),
             ),
             TimeBiasConfig(
@@ -151,7 +138,7 @@ my_config = [
 # Instantiate all of our plugins
 controller = SimpleControllerPlugin('Cobra Simple Controller Plugin')
 plugins = [
-    LcmLogTransportPlugin('Cobra LCM Log Transport Plugin'),
+    Aspn23RosTransportPlugin('Cobra ASPN23-ROS Transport Plugin'),
     EkfFusionStrategyPlugin('Cobra EKF Fusion Strategy Plugin'),
     StandardFusionPlugin('Cobra Standard Fusion Plugin'),
     StandardGpsInsStateModelingPlugin('Cobra Standard State Modeling Plugin'),
@@ -163,7 +150,7 @@ plugins = [
         'Cobra Standard Logging Plugin',
         global_log_level=LoggingLevel.INFO,  # Switch to `DEBUG` for more informative log output
     ),
-    StandardRegistryPlugin('Cobra Standard Registry Plugin', config=my_config),
+    StandardRegistryPlugin('Cobra Standard Registry Plugin', config=my_config),  # type: ignore[arg-type]
     StandardPreprocessorPlugin('Cobra Standard Preprocessor Plugin'),
     StandardOrchestrationPlugin('Cobra Standard Orchestration Plugin'),
 ]
