diff --git a/apps/standard/gps_ins.py b/apps/standard/lcm_relay.py
index f41b96bb..35af88d4 100755
--- a/apps/standard/gps_ins.py
+++ b/apps/standard/lcm_relay.py
@@ -1,7 +1,5 @@
 #!/usr/bin/env python3
 
-import sys
-
 import numpy as np
 
 # API imports
@@ -10,7 +8,7 @@ from pntos.api import EstimateWithCovariance, EstimateWithCovarianceType, Loggin
 # Import Cobra plugins and config structs
 from pntos.cobra import (
     EkfFusionStrategyPlugin,
-    LcmLogTransportPlugin,
+    LcmTransportPlugin,
     ManualHeadingAlignInitializationPlugin,
     SimpleControllerPlugin,
     StandardFusionPlugin,
@@ -28,7 +26,7 @@ from pntos.cobra.config import (
     ImuConfig,
     ImuRotatorConfig,
     InertialConfig,
-    LcmLogTransportConfig,
+    LcmTransportConfig,
     ManualHeadingAlignmentConfig,
     PinsonStateBlockConfig,
     SensorConfig,
@@ -36,9 +34,6 @@ from pntos.cobra.config import (
     StandardOrchestrationConfig,
     TimeAdjusterConfig,
 )
-from pntos_python_datasets import EXAMPLE_LCM_LOG
-
-OUTPUT_LOG = sys.argv[1] if len(sys.argv) > 1 else 'pntos_output.log'
 
 # Config setup
 C_imu_to_platform = (
@@ -56,12 +51,7 @@ imu_model = ImuConfig(
     gyro_random_walk_sigma=(9.9e-4, 9.9e-4, 6.7e-5),
 )
 my_config = [
-    LcmLogTransportConfig(
-        input_file=EXAMPLE_LCM_LOG,
-        output_file=OUTPUT_LOG,
-        output_version=AspnVersion.V23,
-        group='config/lcm_log_transport',
-    ),
+    LcmTransportConfig(output_version=AspnVersion.V23, group='config/lcm_transport'),
     StandardOrchestrationConfig(
         best_sol_channel='/solution/pntos/pva',
         imu_sol_channel='/solution/pntos-imu/pva',
@@ -140,7 +130,7 @@ my_config = [
 # Instantiate all of our plugins
 controller = SimpleControllerPlugin('Cobra Simple Controller Plugin')
 plugins = [
-    LcmLogTransportPlugin('Cobra LCM Log Transport Plugin'),
+    LcmTransportPlugin('Cobra LCM Transport Plugin'),
     EkfFusionStrategyPlugin('Cobra EKF Fusion Strategy Plugin'),
     StandardFusionPlugin('Cobra Standard Fusion Plugin'),
     StandardGpsInsStateModelingPlugin('Cobra Standard State Modeling Plugin'),
