diff --git a/apps/tutorial/gps_ins.py b/apps/tutorial/gps_vel_ins.py
index 933b9c86..c90f958e 100755
--- a/apps/tutorial/gps_ins.py
+++ b/apps/tutorial/gps_vel_ins.py
@@ -16,7 +16,7 @@ from pntos.cobra import (
     StandardPreprocessorPlugin,
     StandardRegistryPlugin,
     TutorialGpsInsStateModelingPlugin,
-    TutorialGpsOrchestrationPlugin,
+    TutorialGpsVelOrchestrationPlugin,
     TutorialInitializationPlugin,
     UiLogPlottingPlugin,
 )
@@ -101,6 +101,7 @@ my_config = [
     TutorialOrchestrationConfig(
         gps_channel='/sensor/ublox-ZED-F9T/position',
         group='config/orchestration',
+        velocity_channel='/sensor/ublox-ZED-F9T/velocity',
     ),
     TimeAdjusterConfig(
         group='config/time_adjuster',
@@ -120,7 +121,10 @@ my_config = [
     ),
     TimeBiasConfig(
         group='config/time_bias',
-        channels_to_correct=('/sensor/ublox-ZED-F9T/position',),
+        channels_to_correct=(
+            '/sensor/ublox-ZED-F9T/position',
+            '/sensor/ublox-ZED-F9T/velocity',
+        ),
         time_bias=int(0.15 * 1e9),
     ),
 ]
@@ -142,7 +146,7 @@ plugins = [
     StandardRegistryPlugin('Cobra Standard Registry Plugin', config=my_config),
     StandardPreprocessorPlugin('Cobra Standard Preprocessor Plugin'),
     UiLogPlottingPlugin('Cobra UI Logfile Plotting Plugin'),
-    TutorialGpsOrchestrationPlugin('Cobra Tutorial Orchestration Plugin'),
+    TutorialGpsVelOrchestrationPlugin('Cobra Tutorial Orchestration Plugin'),
 ]
 
 # Start the controller, and pass it all of the other plugins to use
