diff --git a/pntos-cobra-apps/src/pntos/apps/standard/pos_ins.py b/pntos-cobra-apps/src/pntos/apps/standard/pos_ins_bodyvel.py
index 9903fdf3..4c6b31e6 100755
--- a/pntos-cobra-apps/src/pntos/apps/standard/pos_ins.py
+++ b/pntos-cobra-apps/src/pntos/apps/standard/pos_ins_bodyvel.py
@@ -29,6 +29,7 @@ from pntos.cobra import (
 from pntos.cobra.config import (
     BaseConfig,
     ControllerConfig,
+    DownsamplerConfig,
     FogmConfig,
     FogmStateBlockConfig,
     FusionEngineConfig,
@@ -37,6 +38,7 @@ from pntos.cobra.config import (
     InertialConfig,
     LcmLogTransportConfig,
     ManualHeadingAlignmentConfig,
+    PinsonBodyVelocityMPConfig,
     PinsonStateBlockConfig,
     PinsonWithNedFogmPositionMPConfig,
     StandardOrchestrationConfig,
@@ -47,7 +49,7 @@ from pntos_python_datasets_lcm import EXAMPLE_LCM_LOG
 
 
 def main() -> None:
-    """Run the pos_ins app."""
+    """Run the pos_ins_bodyvel app."""
     OUTPUT_LOG = sys.argv[1] if len(sys.argv) > 1 else 'pntos_output.log'
 
     # Config setup
@@ -74,6 +76,7 @@ def main() -> None:
             channels_to_process=(
                 '/sensor/vn-100/imu',
                 '/sensor/ublox-ZED-F9T/position',
+                '/sensor/simulated/velocity',
             ),
         ),
         ControllerConfig(),
@@ -111,6 +114,14 @@ def main() -> None:
                     state_block_labels=('pinson15', 'pos_sensor_error'),
                     lever_arm=(-0.50, 0.38, -0.05),
                 ),
+                PinsonBodyVelocityMPConfig(
+                    group='config/bodyvel_measurement_processor',
+                    label='body_vel',
+                    channel='/sensor/simulated/velocity',
+                    state_block_labels=('pinson15',),
+                    lever_arm=(0.0, 0.0, 0.0),
+                    orientation=(1.0, 0.0, 0.0, 0.0),
+                ),
             ),
             inertial_config=InertialConfig(
                 group='config/inertial',
@@ -142,6 +153,11 @@ def main() -> None:
                     channels=('/sensor/ublox-ZED-F9T/position',),
                     time_bias=int(0.15 * 1e9),
                 ),
+                DownsamplerConfig(
+                    group='config/downsampler',
+                    channels=('/sensor/simulated/velocity',),
+                    downsampling_factors=(10,),  # Downsample from 1 Hz to 0.1 Hz
+                ),
             ),
         ),
     ]
