diff --git a/apps/standard/pos_ins.py b/apps/standard/outage_sim.py
index 701f7e6b..03c9c7ee 100755
--- a/apps/standard/pos_ins.py
+++ b/apps/standard/outage_sim.py
@@ -23,6 +23,7 @@ from pntos.cobra import (
 )
 from pntos.cobra.config import (
     AspnVersion,
+    BarometerToAltitudeConfig,
     ControllerConfig,
     FogmConfig,
     FogmStateBlockConfig,
@@ -32,6 +33,8 @@ from pntos.cobra.config import (
     InertialConfig,
     LcmLogTransportConfig,
     ManualHeadingAlignmentConfig,
+    MeasurementProcessorConfig,
+    OutageConfig,
     PinsonStateBlockConfig,
     SensorConfig,
     SensorMeasurementProcessorConfig,
@@ -68,6 +71,8 @@ my_config = [
         channels_to_process=(
             '/sensor/vn-100/imu',
             '/sensor/ublox-ZED-F9T/position',
+            # '/sensor/bmp388/baro_pressure',   # uncomment me to use a baro update
+            # '/sensor/ublox-ZED-F9T/velocity', # uncomment me to use a velocity update
         ),
     ),
     ControllerConfig(),
@@ -96,6 +101,20 @@ my_config = [
                     tau=(300.0, 300.0, 200.0),
                 ),
             ),
+            FogmStateBlockConfig(
+                group='config/alt_fogm_block',
+                label='alt_fogm',
+                estimate_with_covariance=EstimateWithCovariance(
+                    type=EstimateWithCovarianceType.EWC_GENERIC,
+                    estimate=np.zeros((1,)),
+                    covariance=(np.array([[100.0**2]])),
+                ),
+                fogm_model=FogmConfig(
+                    group='config/alt_sensor_error',
+                    sigma=(100.0,),
+                    tau=(3600.0,),
+                ),
+            ),
         ),
         mp_configs=(
             SensorMeasurementProcessorConfig(
@@ -112,6 +131,28 @@ my_config = [
                     sensor_name='position',
                 ),
             ),
+            SensorMeasurementProcessorConfig(
+                group='config/alt_measurement_processor',
+                identifier='pinson_altitude',
+                label='alt',
+                channel='/sensor/bmp388/altitude',
+                state_block_labels=('pinson15', 'alt_fogm'),
+                aux_channels=('INERTIAL_PVA',),
+                sensor_config=SensorConfig(
+                    group='config/alt_state_modeling',
+                    lever_arm=(0.0, 0.0, 0.0),
+                    orientation=(0.0, 0.0, 0.0, 0.0),
+                    sensor_name='altitude',
+                ),
+            ),
+            MeasurementProcessorConfig(
+                group='config/vel_measurement_processor',
+                identifier='pinson_velocity',
+                label='vel',
+                channel='/sensor/ublox-ZED-F9T/velocity',
+                state_block_labels=('pinson15',),
+                aux_channels=('INERTIAL_PVA',),
+            ),
         ),
         inertial_config=InertialConfig(
             group='config/inertial',
@@ -140,10 +181,25 @@ my_config = [
             ),
             TimeBiasConfig(
                 group='config/time_bias',
-                channels_to_correct=('/sensor/ublox-ZED-F9T/position',),
+                channels_to_correct=(
+                    '/sensor/ublox-ZED-F9T/position',
+                    '/sensor/ublox-ZED-F9T/velocity',
+                ),
                 time_bias=int(0.15 * 1e9),
             ),
+            BarometerToAltitudeConfig(
+                group='config/pressure_to_alt',
+                channel='/sensor/bmp388/baro_pressure',
+                alt_sigma=30.0,
+            ),
+            OutageConfig(
+                group='config/pos_outage',
+                channel='/sensor/ublox-ZED-F9T/position',
+                start_time=1000.0,
+                end_time=1600.0,
+            ),
         ),
+        max_prop_interval=1.0,
     ),
 ]
 # End Config
