diff --git a/apps/standard/lcm_relay.py b/apps/advanced/ui.py
index c4bbc3fb..9732d59c 100755
--- a/apps/standard/lcm_relay.py
+++ b/apps/advanced/ui.py
@@ -7,7 +7,9 @@ from pntos.api import EstimateWithCovariance, EstimateWithCovarianceType, Loggin
 
 # Import Cobra plugins and config structs
 from pntos.cobra import (
+    CobraUiLogPlayerPlugin,
     EkfFusionStrategyPlugin,
+    ExperimentalCobraUiPlugin,
     LcmTransportPlugin,
     ManualHeadingAlignInitializationPlugin,
     StandardControllerPlugin,
@@ -21,6 +23,7 @@ from pntos.cobra import (
 )
 from pntos.cobra.config import (
     ControllerConfig,
+    ExperimentalCobraUiConfig,
     FogmConfig,
     FogmStateBlockConfig,
     FusionEngineConfig,
@@ -123,6 +126,9 @@ my_config = [
             ),
         ),
     ),
+    ExperimentalCobraUiConfig(
+        group='config/cobra_ui',
+    ),
 ]
 # End Config
 
@@ -144,6 +150,8 @@ plugins = [
     StandardRegistryPlugin('Cobra Standard Registry Plugin', config=my_config),
     StandardPreprocessorPlugin('Cobra Standard Preprocessor Plugin'),
     StandardOrchestrationPlugin('Cobra Standard Orchestration Plugin'),
+    ExperimentalCobraUiPlugin('Experimental Cobra UI Plugin'),
+    CobraUiLogPlayerPlugin('Cobra UI Log Player Plugin'),
 ]
 
 # Start the controller, and pass it all of the other plugins to use
