diff --git a/apps/tutorial/gps_ins.py b/apps/tutorial/gps_vel_ins.py
index ceb1f9e3..c224b0e4 100755
--- a/apps/tutorial/gps_ins.py
+++ b/apps/tutorial/gps_vel_ins.py
@@ -16,7 +16,7 @@ from pntos.cobra import (
     StandardLoggingPlugin,
     StandardPreprocessorPlugin,
     StandardRegistryPlugin,
-    TutorialGpsOrchestrationPlugin,
+    TutorialGpsVelOrchestrationPlugin,
     TutorialInitializationPlugin,
 )
 from pntos.cobra.config import (
@@ -96,6 +96,7 @@ my_config = [
     TutorialOrchestrationConfig(
         gps_channel='/sensor/ublox-ZED-F9T/position',
         group='config/orchestration',
+        velocity_channel='/sensor/ublox-ZED-F9T/velocity',
     ),
     TimeAdjusterConfig(
         group='config/time_adjuster',
@@ -127,7 +128,7 @@ plugins = [
     ),
     StandardRegistryPlugin('Cobra Standard Registry Plugin', config=my_config),
     StandardPreprocessorPlugin('Cobra Standard Preprocessor Plugin'),
-    TutorialGpsOrchestrationPlugin('Cobra Tutorial Orchestration Plugin'),
+    TutorialGpsVelOrchestrationPlugin('Cobra Tutorial Orchestration Plugin'),
 ]
 
 # Start the controller, and pass it all of the other plugins to use
