diff --git a/apps/standard/pos_ins.py b/apps/standard/pos_ins_bodyvel.py
index 86eea2f3..f751a0df 100755
--- a/apps/standard/pos_ins.py
+++ b/apps/standard/pos_ins_bodyvel.py
@@ -32,6 +32,7 @@ from pntos.cobra.config import (
     InertialConfig,
     LcmLogTransportConfig,
     ManualHeadingAlignmentConfig,
+    PinsonBodyVelocityMPConfig,
     PinsonStateBlockConfig,
     PinsonWithNedFogmPositionMPConfig,
     StandardOrchestrationConfig,
@@ -67,6 +68,7 @@ my_config = [
         channels_to_process=(
             '/sensor/vn-100/imu',
             '/sensor/ublox-ZED-F9T/position',
+            '/sensor/simulated/velocity',
         ),
     ),
     ControllerConfig(),
@@ -104,6 +106,14 @@ my_config = [
                 state_block_labels=('pinson15', 'pos_sensor_error'),
                 lever_arm=(-0.50, 0.38, -0.05),
             ),
+            PinsonBodyVelocityMPConfig(
+                group='config/bodyvel_measurement_processor',
+                label='body_vel',
+                channel='/sensor/simulated/velocity',
+                state_block_labels=('pinson15',),
+                lever_arm=(0.0, 0.0, 0.0),
+                orientation=(0.0, 0.0, 0.0, 0.0),
+            ),
         ),
         inertial_config=InertialConfig(
             group='config/inertial',
