diff --git a/apps/tutorial/gps_ins.py b/apps/tutorial/gps_vel_ins.py
index e2e410f8..460d98c5 100755
--- a/apps/tutorial/gps_ins.py
+++ b/apps/tutorial/gps_vel_ins.py
@@ -16,7 +16,7 @@ from pntos.cobra import (
     StandardLoggingPlugin,
     StandardPreprocessorPlugin,
     StandardRegistryPlugin,
-    TutorialGpsOrchestrationPlugin,
+    TutorialGpsVelOrchestrationPlugin,
     TutorialInitializationPlugin,
     UiLogPlottingPlugin,
 )
@@ -98,6 +98,7 @@ my_config = [
     TutorialOrchestrationConfig(
         gps_channel='/sensor/ublox-ZED-F9T/position',
         group='config/orchestration',
+        velocity_channel='/sensor/ublox-ZED-F9T/velocity',
     ),
     TimeAdjusterConfig(
         group='config/time_adjuster',
@@ -136,7 +137,7 @@ plugins = [
     StandardRegistryPlugin('Cobra Standard Registry Plugin', config=my_config),
     StandardPreprocessorPlugin('Cobra Standard Preprocessor Plugin'),
     UiLogPlottingPlugin('Cobra UI Logfile Plotting Plugin'),
-    TutorialGpsOrchestrationPlugin('Cobra Tutorial Orchestration Plugin'),
+    TutorialGpsVelOrchestrationPlugin('Cobra Tutorial Orchestration Plugin'),
 ]
 
 # Start the controller, and pass it all of the other plugins to use
