diff --git a/apps/standard/pos_ins.py b/apps/standard/direction_to_points.py
index 9903fdf3..d6a71d9b 100755
--- a/apps/standard/pos_ins.py
+++ b/apps/standard/direction_to_points.py
@@ -29,6 +29,7 @@ from pntos.cobra import (
 from pntos.cobra.config import (
     BaseConfig,
     ControllerConfig,
+    Direction3dToPointsMPConfig,
     FogmConfig,
     FogmStateBlockConfig,
     FusionEngineConfig,
@@ -38,7 +39,6 @@ from pntos.cobra.config import (
     LcmLogTransportConfig,
     ManualHeadingAlignmentConfig,
     PinsonStateBlockConfig,
-    PinsonWithNedFogmPositionMPConfig,
     StandardOrchestrationConfig,
     TimeAdjusterConfig,
     TimeBiasConfig,
@@ -47,7 +47,7 @@ from pntos_python_datasets_lcm import EXAMPLE_LCM_LOG
 
 
 def main() -> None:
-    """Run the pos_ins app."""
+    """Run the direction_to_points app."""
     OUTPUT_LOG = sys.argv[1] if len(sys.argv) > 1 else 'pntos_output.log'
 
     # Config setup
@@ -74,6 +74,7 @@ def main() -> None:
             channels_to_process=(
                 '/sensor/vn-100/imu',
                 '/sensor/ublox-ZED-F9T/position',
+                '/sensor/simulated/directiontoknownfeature',
             ),
         ),
         ControllerConfig(),
@@ -104,12 +105,13 @@ def main() -> None:
                 ),
             ),
             mp_configs=(
-                PinsonWithNedFogmPositionMPConfig(
-                    group='config/pos_measurement_processor',
-                    label='pos',
-                    channel='/sensor/ublox-ZED-F9T/position',
-                    state_block_labels=('pinson15', 'pos_sensor_error'),
-                    lever_arm=(-0.50, 0.38, -0.05),
+                Direction3dToPointsMPConfig(
+                    group='config/direction3D_to_points_measurement_processor',
+                    label='dir3D_to_points',
+                    channel='/sensor/simulated/directiontoknownfeature',
+                    state_block_labels=('pinson15',),
+                    lever_arm=(0.80, 0.0, 0.05),
+                    orientation=(0.707106781, 0.0, 0.707106781, 0.0),
                 ),
             ),
             inertial_config=InertialConfig(
