diff --git a/apps/standard/pos_ins.py b/apps/standard/direction_to_points.py
index 86eea2f3..15873aa9 100755
--- a/apps/standard/pos_ins.py
+++ b/apps/standard/direction_to_points.py
@@ -24,6 +24,8 @@ from pntos.cobra import (
 from pntos.cobra.config import (
     AspnVersion,
     ControllerConfig,
+    Direction3dToPointsMPConfig,
+    DownsamplerConfig,
     FogmConfig,
     FogmStateBlockConfig,
     FusionEngineConfig,
@@ -33,7 +35,6 @@ from pntos.cobra.config import (
     LcmLogTransportConfig,
     ManualHeadingAlignmentConfig,
     PinsonStateBlockConfig,
-    PinsonWithNedFogmPositionMPConfig,
     StandardOrchestrationConfig,
     TimeAdjusterConfig,
     TimeBiasConfig,
@@ -67,6 +68,7 @@ my_config = [
         channels_to_process=(
             '/sensor/vn-100/imu',
             '/sensor/ublox-ZED-F9T/position',
+            '/sensor/simulated/directiontoknownfeature',
         ),
     ),
     ControllerConfig(),
@@ -97,12 +99,13 @@ my_config = [
             ),
         ),
         mp_configs=(
-            PinsonWithNedFogmPositionMPConfig(
-                group='config/pos_measurement_processor',
-                label='pos',
-                channel='/sensor/ublox-ZED-F9T/position',
-                state_block_labels=('pinson15', 'pos_sensor_error'),
-                lever_arm=(-0.50, 0.38, -0.05),
+            Direction3dToPointsMPConfig(
+                group='config/direction3D_to_points_measurement_processor',
+                label='dir3D_to_points',
+                channel='/sensor/simulated/directiontoknownfeature',
+                state_block_labels=('pinson15',),
+                lever_arm=(0.80, 0.0, 0.05),
+                orientation=(0.707106781, 0.0, 0.707106781, 0.0),
             ),
         ),
         inertial_config=InertialConfig(
@@ -135,6 +138,11 @@ my_config = [
                 channels_to_correct=('/sensor/ublox-ZED-F9T/position',),
                 time_bias=int(0.15 * 1e9),
             ),
+            DownsamplerConfig(
+                group='config/downsampler',
+                channels_to_downsample=('/sensor/simulated/directiontoknownfeature',),
+                downsampling_factors=(1,),
+            ),
         ),
     ),
 ]
