diff --git a/pntos-cobra-apps/src/pntos/apps/standard/pos_ins.py b/pntos-cobra-apps/src/pntos/apps/standard/pos_vel_ins.py
index 9903fdf3..28bbae68 100755
--- a/pntos-cobra-apps/src/pntos/apps/standard/pos_ins.py
+++ b/pntos-cobra-apps/src/pntos/apps/standard/pos_vel_ins.py
@@ -1,5 +1,12 @@
 #!/usr/bin/env python3
 
+"""App to process IMU, position, and velocity measurements.
+
+
+Note: This app uses a simpler position measurement model than some of the other apps.
+Thus, the filter solution is less accurate than those apps.
+"""
+
 import sys
 
 import numpy as np
@@ -37,8 +44,9 @@ from pntos.cobra.config import (
     InertialConfig,
     LcmLogTransportConfig,
     ManualHeadingAlignmentConfig,
+    PinsonPositionMPConfig,
     PinsonStateBlockConfig,
-    PinsonWithNedFogmPositionMPConfig,
+    PinsonVelocityMPConfig,
     StandardOrchestrationConfig,
     TimeAdjusterConfig,
     TimeBiasConfig,
@@ -47,7 +55,7 @@ from pntos_python_datasets_lcm import EXAMPLE_LCM_LOG
 
 
 def main() -> None:
-    """Run the pos_ins app."""
+    """Run the pos_vel_ins app."""
     OUTPUT_LOG = sys.argv[1] if len(sys.argv) > 1 else 'pntos_output.log'
 
     # Config setup
@@ -74,6 +82,7 @@ def main() -> None:
             channels_to_process=(
                 '/sensor/vn-100/imu',
                 '/sensor/ublox-ZED-F9T/position',
+                '/sensor/ublox-ZED-F9T/velocity',
             ),
         ),
         ControllerConfig(),
@@ -104,13 +113,19 @@ def main() -> None:
                 ),
             ),
             mp_configs=(
-                PinsonWithNedFogmPositionMPConfig(
+                PinsonPositionMPConfig(
                     group='config/pos_measurement_processor',
                     label='pos',
                     channel='/sensor/ublox-ZED-F9T/position',
-                    state_block_labels=('pinson15', 'pos_sensor_error'),
+                    state_block_labels=('pinson15',),
                     lever_arm=(-0.50, 0.38, -0.05),
                 ),
+                PinsonVelocityMPConfig(
+                    group='config/vel_measurement_processor',
+                    label='vel',
+                    channel='/sensor/ublox-ZED-F9T/velocity',
+                    state_block_labels=('pinson15',),
+                ),
             ),
             inertial_config=InertialConfig(
                 group='config/inertial',
@@ -139,7 +154,10 @@ def main() -> None:
                 ),
                 TimeBiasConfig(
                     group='config/time_bias',
-                    channels=('/sensor/ublox-ZED-F9T/position',),
+                    channels=(
+                        '/sensor/ublox-ZED-F9T/position',
+                        '/sensor/ublox-ZED-F9T/velocity',
+                    ),
                     time_bias=int(0.15 * 1e9),
                 ),
             ),
