diff --git a/apps/standard/lcm_relay.py b/apps/advanced/ui.py
index 31982bb4..ac5de308 100755
--- a/apps/standard/lcm_relay.py
+++ b/apps/advanced/ui.py
@@ -12,7 +12,9 @@ from pntos.api import (
 
 # Import Cobra plugins and config structs
 from pntos.cobra import (
+    CobraUiLogPlayerPlugin,
     EkfFusionStrategyPlugin,
+    ExperimentalCobraUiPlugin,
     LcmTransportPlugin,
     ManualHeadingAlignInitializationPlugin,
     StandardControllerPlugin,
@@ -27,6 +29,7 @@ from pntos.cobra import (
 from pntos.cobra.config import (
     BaseConfig,
     ControllerConfig,
+    ExperimentalCobraUiConfig,
     FogmConfig,
     FogmStateBlockConfig,
     FusionEngineConfig,
@@ -129,6 +132,9 @@ my_config: list[BaseConfig] = [
             ),
         ),
     ),
+    ExperimentalCobraUiConfig(
+        group='config/cobra_ui',
+    ),
 ]
 # End Config
 
@@ -150,6 +156,8 @@ plugins: list[CommonPlugin] = [
     StandardRegistryPlugin('Cobra Standard Registry Plugin', config=my_config),
     StandardPreprocessorPlugin('Cobra Standard Preprocessor Plugin'),
     StandardOrchestrationPlugin('Cobra Standard Orchestration Plugin'),
+    ExperimentalCobraUiPlugin('Experimental Cobra UI Plugin'),
+    CobraUiLogPlayerPlugin('Cobra UI Log Player Plugin'),
 ]
 
 # Start the controller, and pass it all of the other plugins to use
