diff --git a/apps/standard/gps_ins.py b/apps/standard/gps_vel_ins.py
index 46f2a3b2..65040937 100755
--- a/apps/standard/gps_ins.py
+++ b/apps/standard/gps_vel_ins.py
@@ -1,5 +1,12 @@
 #!/usr/bin/env python3
 
+"""App to process IMU, position, and velocity measurements.
+
+
+Note: This app uses a simpler position measurement model than some of the other apps.
+Thus, the filter solution is less accurate than those apps.
+"""
+
 import sys
 
 import numpy as np
@@ -31,6 +38,7 @@ from pntos.cobra.config import (
     InertialConfig,
     LcmLogTransportConfig,
     ManualHeadingAlignmentConfig,
+    MeasurementProcessorConfig,
     PinsonStateBlockConfig,
     SensorConfig,
     SensorMeasurementProcessorConfig,
@@ -68,6 +76,7 @@ my_config = [
         channels_to_process=(
             '/sensor/vn-100/imu',
             '/sensor/ublox-ZED-F9T/position',
+            '/sensor/ublox-ZED-F9T/velocity',
         ),
     ),
     ControllerConfig(group='controller'),
@@ -101,10 +110,10 @@ my_config = [
         mp_configs=(
             SensorMeasurementProcessorConfig(
                 group='config/gps_measurement_processor',
-                identifier='pinson_with_ned_fogm_position',
+                identifier='pinson_position',
                 label='gps',
                 channel='/sensor/ublox-ZED-F9T/position',
-                state_block_labels=('pinson15', 'pos_sensor_error'),
+                state_block_labels=('pinson15',),
                 aux_channels=('INERTIAL_PVA',),
                 sensor_config=SensorConfig(
                     group='config/gp3d_state_modeling',
@@ -113,6 +122,14 @@ my_config = [
                     sensor_name='position',
                 ),
             ),
+            MeasurementProcessorConfig(
+                group='config/vel_measurement_processor',
+                identifier='pinson_velocity',
+                label='vel',
+                channel='/sensor/ublox-ZED-F9T/velocity',
+                state_block_labels=('pinson15',),
+                aux_channels=('INERTIAL_PVA',),
+            ),
         ),
         inertial_config=InertialConfig(
             group='config/inertial',
@@ -144,7 +161,10 @@ my_config = [
             TimeBiasConfig(
                 group='config/time_bias',
                 identifier='time_bias',
-                channels_to_correct=('/sensor/ublox-ZED-F9T/position',),
+                channels_to_correct=(
+                    '/sensor/ublox-ZED-F9T/position',
+                    '/sensor/ublox-ZED-F9T/velocity',
+                ),
                 time_bias=int(0.15 * 1e9),
             ),
         ),
