diff --git a/pntos-cobra-apps/src/pntos/apps/standard/pos_vel_ins.py b/pntos-cobra-apps/src/pntos/apps/standard/posvel_ins.py
index 28bbae68..b96865b9 100755
--- a/pntos-cobra-apps/src/pntos/apps/standard/pos_vel_ins.py
+++ b/pntos-cobra-apps/src/pntos/apps/standard/posvel_ins.py
@@ -1,11 +1,13 @@
 #!/usr/bin/env python3
 
-"""App to process IMU, position, and velocity measurements.
-
+"""App to process IMU, position and velocity measurements.
 
 Note: This app uses a simpler position measurement model than some of the other apps.
 Thus, the filter solution is less accurate than those apps.
-"""
+
+Note: This is the same data as the pos_vel_ins tutorial app, but with the position
+and NED velocity measurements combined into a PVA measurement to be processed as a
+single measurement."""
 
 import sys
 
@@ -44,9 +46,8 @@ from pntos.cobra.config import (
     InertialConfig,
     LcmLogTransportConfig,
     ManualHeadingAlignmentConfig,
-    PinsonPositionMPConfig,
     PinsonStateBlockConfig,
-    PinsonVelocityMPConfig,
+    PosVelMPConfig,
     StandardOrchestrationConfig,
     TimeAdjusterConfig,
     TimeBiasConfig,
@@ -55,7 +56,7 @@ from pntos_python_datasets_lcm import EXAMPLE_LCM_LOG
 
 
 def main() -> None:
-    """Run the pos_vel_ins app."""
+    """Run the posvel_ins app."""
     OUTPUT_LOG = sys.argv[1] if len(sys.argv) > 1 else 'pntos_output.log'
 
     # Config setup
@@ -82,7 +83,7 @@ def main() -> None:
             channels_to_process=(
                 '/sensor/vn-100/imu',
                 '/sensor/ublox-ZED-F9T/position',
-                '/sensor/ublox-ZED-F9T/velocity',
+                '/sensor/ublox-ZED-F9T/pva',
             ),
         ),
         ControllerConfig(),
@@ -113,19 +114,13 @@ def main() -> None:
                 ),
             ),
             mp_configs=(
-                PinsonPositionMPConfig(
-                    group='config/pos_measurement_processor',
-                    label='pos',
-                    channel='/sensor/ublox-ZED-F9T/position',
+                PosVelMPConfig(
+                    group='config/posvel_measurement_processor',
+                    label='posvel',
+                    channel='/sensor/ublox-ZED-F9T/pva',
                     state_block_labels=('pinson15',),
                     lever_arm=(-0.50, 0.38, -0.05),
                 ),
-                PinsonVelocityMPConfig(
-                    group='config/vel_measurement_processor',
-                    label='vel',
-                    channel='/sensor/ublox-ZED-F9T/velocity',
-                    state_block_labels=('pinson15',),
-                ),
             ),
             inertial_config=InertialConfig(
                 group='config/inertial',
@@ -154,10 +149,7 @@ def main() -> None:
                 ),
                 TimeBiasConfig(
                     group='config/time_bias',
-                    channels=(
-                        '/sensor/ublox-ZED-F9T/position',
-                        '/sensor/ublox-ZED-F9T/velocity',
-                    ),
+                    channels=('/sensor/ublox-ZED-F9T/pva',),
                     time_bias=int(0.15 * 1e9),
                 ),
             ),
