diff --git a/apps/fusion_gps_vel_ins.py b/apps/fusion_gps_ins.py
index b79c5d19..1e63de5e 100755
--- a/apps/fusion_gps_vel_ins.py
+++ b/apps/fusion_gps_ins.py
@@ -11,7 +11,7 @@ from pntos.cobra import (
     SimpleEkfFusionStrategyPlugin,
     SimpleFusionPlugin,
     SimpleGpsInsStateModelingPlugin,
-    SimpleGpsVelOrchestrationPlugin,
+    SimpleGpsOrchestrationPlugin,
     SimpleInertialPlugin,
     SimpleInitializationPlugin,
     SimpleLoggingPlugin,
@@ -82,7 +82,6 @@ my_config = [
     OrchestrationConfig(
         gps_channel='/sensor/ublox-ZED-F9T/position',
         group='config/orchestration',
-        velocity_channel='/sensor/ublox-ZED-F9T/velocity',
     ),
     TimeAdjusterConfig(
         group='config/time_adjuster',
@@ -105,7 +104,7 @@ plugins = [
         'Cobra Simple Logging Plugin',
         global_log_level=LoggingLevel.INFO,  # Switch to `DEBUG` for more informative log output
     ),
-    SimpleGpsVelOrchestrationPlugin('Cobra Simple Orchestration Plugin'),
+    SimpleGpsOrchestrationPlugin('Cobra Simple Orchestration Plugin'),
     SimpleRegistryPlugin('Cobra Simple Registry Plugin', config=my_config),
     SimpleCobraPreprocessorPlugin('Cobra Simple Preprocessor Plugin'),
 ]
