diff --git a/pntos-cobra/src/pntos/cobra/standard_plugins//controller/StandardControllerPlugin.py b/pntos-cobra/src/pntos/cobra/advanced_plugins//buscat/BuscatControllerPlugin.py
index f2a2d9204..b2a841155 100644
--- a/pntos-cobra/src/pntos/cobra/standard_plugins//controller/StandardControllerPlugin.py
+++ b/pntos-cobra/src/pntos/cobra/advanced_plugins//buscat/BuscatControllerPlugin.py
@@ -7,59 +7,59 @@ from pntos.api import (
     LoggingLevel,
     LoggingPlugin,
     Mediator,
-    OrchestrationPlugin,
     RegistryPlugin,
     TransportPlugin,
     UiPlugin,
 )
-from pntos.cobra.config import ControllerConfig, config_from_registry
+from pntos.cobra.config import BuscatConfig, config_from_registry
 from pntos.cobra.utils import (
     SortedPlugins,
-    UiMediatorInterface,
     find_base_plugin_type,
     print_message,
     sort_plugins_dataclass,
     validate_plugins,
 )
 
-from .StandardMediator import StandardMediator
-from .StandardMessageStreamConfig import StandardMessageStreamConfig
+from .BuscatMediator import BuscatMediator
 
 
-class StandardControllerPlugin(ControllerPlugin):
+class BuscatControllerPlugin(ControllerPlugin):
     """
-    This is a simple single-threaded controller plugin.
+    This is a simple single-threaded Buscat controller plugin.
+
+    The purpose of this plugin is to route one or more data streams from
+    :class:`TransportPlugins <pntos.api.TransportPlugin>` back out through one or more
+    :class:`TransportPlugins <pntos.api.TransportPlugin>`, as specified in the supplied
+    :attr:`BuscatConfig.output_transports <pntos.cobra.config.BuscatConfig.output_transports>`.
+    This has the effect of combining multiple data streams and converting all input to the formats
+    supported by the output :class:`TransportPlugins <pntos.api.TransportPlugin>`. Note that this
+    controller does not do any buffering or sorting of the input data before publishing. It (or more
+    specifically, the :class:`BuscatMediator <pntos.cobra.internal.BuscatMediator>`) also does not
+    explicitly pass input data to plugins other than :class:`TransportPlugins <pntos.api.TransportPlugin>`,
+    and thus will not support sensor fusion without some modification.
 
     Here are the plugins and corresponding expected number of instances this controller
     looks for:
 
-    - FusionPlugin - at least 1
-    - FusionStrategyPlugin - at least 1
-    - InertialPlugin - at least 1
-    - InitializationPlugin - at least 1
     - LoggingPlugin - 1
-    - OrchestrationPlugin - 1
     - RegistryPlugin - 1
     - TransportPlugin - at least 1
     - UiPlugin - any, but only 1 can require the main thread
 
     It checks for the expected plugins, sets up mediators, and then passes the mediators
-    to each plugin in :meth:`pntos.api.CommonPlugin.init_plugin`, initializes the
-    orchestration plugin, then passes off to the :meth:`_main` function.
+    to each plugin in :meth:`pntos.api.CommonPlugin.init_plugin`, then passes off to the
+    :meth:`_main` function.
 
     Inside the main function, calls :meth:`pntos.api.TransportPlugin.start_listening` on
     all transport plugins, checks if the UI needs to update (and passes it the thread if
     so), then waits for a ctrl+c to exit pntOS.
-
-    The TransportPlugins will initiate the necessary filtering through their calls to
-    :meth:`pntos.api.Mediator.process_pntos_message`.
     """
 
     _plugin_resources_location: str | None
 
     def __init__(self, identifier: str) -> None:
         """
-        Cobra Standard Controller
+        Cobra Buscat Controller
 
         Args:
             identifier (str): The plugin identifier passed to the
@@ -112,22 +112,16 @@ class StandardControllerPlugin(ControllerPlugin):
             self._registry_plugin,
             self._logging_plugin,
             self._transport_plugins,
-            orchestration_plugin,
             self._ui_plugins,
-            plugins_for_orchestration,
         ) = self._sort_and_validate_plugins(plugins)
 
         # Hand mediators controller plugin so that mediators can call "shutdown_plugin"
         # on error.
-        StandardMediator._controller_plugin = self
-
-        # initialize stream config and hand to mediator
-        stream_config = StandardMessageStreamConfig()
-        StandardMediator._stream_config = stream_config
+        BuscatMediator._controller_plugin = self
 
         # Create separate mediators to pass to each plugin
         mediators = [
-            StandardMediator(p.identifier, find_base_plugin_type(p))
+            BuscatMediator(p.identifier, find_base_plugin_type(p))
             for p in self._plugins
         ]
 
@@ -154,8 +148,7 @@ class StandardControllerPlugin(ControllerPlugin):
         )
 
         # Give mediators a registry
-        StandardMediator.registry = self._registry_plugin.new_registry(initial_config)
-        StandardMediator._ui_interface = UiMediatorInterface(StandardMediator.registry)
+        BuscatMediator.registry = self._registry_plugin.new_registry(initial_config)
 
         # Initialize logger second
         assert self._logging_plugin is not None
@@ -168,7 +161,7 @@ class StandardControllerPlugin(ControllerPlugin):
         )
 
         # Give mediators a logging plugin
-        StandardMediator._logging_plugin = self._logging_plugin
+        BuscatMediator._logging_plugin = self._logging_plugin
 
         # call init_plugin() on all the other plugins
         for i, plugin in enumerate(self._plugins):
@@ -182,46 +175,28 @@ class StandardControllerPlugin(ControllerPlugin):
             )
 
         # Give the mediators other needed plugins
-        StandardMediator._transport_plugins.extend(self._transport_plugins)
-        StandardMediator._orchestration_plugin = orchestration_plugin
-
-        # Give the orchestration the plugins it needs
-        orchestration_plugin.init_orchestration_plugin(
-            plugins_for_orchestration, stream_config
-        )
+        BuscatMediator._transport_plugins.extend(self._transport_plugins)
 
-        # Extract controller-specific config
-        temp_mediator = StandardMediator(self.identifier, ControllerPlugin)
-        config = config_from_registry(
-            ControllerConfig, temp_mediator, ControllerConfig.group
-        )
+        # Set the mediators' output transport
+        temp_mediator = BuscatMediator(self.identifier, ControllerPlugin)
+        config = config_from_registry(BuscatConfig, temp_mediator, BuscatConfig.group)
         if config is None:
             self._log(
                 LoggingLevel.ERROR,
-                'Could not extract ControllerConfig from group "controller". Cannot initialize controller plugin.',
+                'Could not extract BuscatConfig from group "buscat". Cannot initialize Buscat controller plugin.',
             )
             return
+        BuscatMediator._output_transports.extend(config.output_transports or ())
 
         with temp_mediator.registry.batch_start('controller/flags') as kvs:
             kvs.request_notify('ready_to_shutdown', self._ready_to_shutdown_callback)
 
-        StandardMediator._buffer_time_nsec = int(config.buffer_length_sec * 1e9)
-        if config.publish_interval is not None:
-            StandardMediator._publish_interval_ns = int(config.publish_interval * 1e9)
-
         # Pass off to main control loop
         self._main()
 
     def _sort_and_validate_plugins(
         self, plugins: list[CommonPlugin]
-    ) -> tuple[
-        RegistryPlugin,
-        LoggingPlugin,
-        list[TransportPlugin],
-        OrchestrationPlugin,
-        list[UiPlugin],
-        list[CommonPlugin],
-    ]:
+    ) -> tuple[RegistryPlugin, LoggingPlugin, list[TransportPlugin], list[UiPlugin]]:
         """
         Utility function to ensure ``plugins`` contains enough plugins to run pntOS
         Cobra. Then assigns and dispatches them to the relevant fields on the
@@ -229,34 +204,21 @@ class StandardControllerPlugin(ControllerPlugin):
         not as expected.
         """
         sorted_plugins: SortedPlugins = sort_plugins_dataclass(plugins)
-        plugins_for_orchestration: list[CommonPlugin] = []
 
         if not validate_plugins(
             sorted_plugins,
             self._log,
             registry_plugins=(1, 1),
             logging_plugins=(1, 1),
-            orchestration_plugins=(1, 1),
             transport_plugins=(1, 1000),
-            fusion_plugins=(1, 1000),
         ):
             raise RuntimeError('Not enough plugins to run pntOS.')
 
-        # Collect plugins to pass to orchestration
-        plugins_for_orchestration.extend(sorted_plugins.fusion_plugins)
-        plugins_for_orchestration.extend(sorted_plugins.fusion_strategy_plugins)
-        plugins_for_orchestration.extend(sorted_plugins.inertial_plugins)
-        plugins_for_orchestration.extend(sorted_plugins.initialization_plugins)
-        plugins_for_orchestration.extend(sorted_plugins.state_modeling_plugins)
-        plugins_for_orchestration.extend(sorted_plugins.preprocessor_plugins)
-
         return (
             sorted_plugins.registry_plugins[0],
             sorted_plugins.logging_plugins[0],
             sorted_plugins.transport_plugins,
-            sorted_plugins.orchestration_plugins[0],
             sorted_plugins.ui_plugins,
-            plugins_for_orchestration,
         )
 
     def _ready_to_shutdown_callback(
@@ -303,10 +265,10 @@ class StandardControllerPlugin(ControllerPlugin):
                 'Press Ctrl + C at any time to shut down pntOS...',
             )
             try:
-                StandardMediator._logging_error_event.wait()
+                BuscatMediator._logging_error_event.wait()
             except KeyboardInterrupt:
                 self._log(LoggingLevel.INFO, 'Keyboard Interrupt Detected.')
 
         self.shutdown_plugin()
-        if StandardMediator._logging_error_event.is_set():
+        if BuscatMediator._logging_error_event.is_set():
             sys.exit(1)
