diff --git a/pntos-cobra/src/pntos/cobra/tutorial_plugins/TutorialPosOrchestrationPlugin.py b/pntos-cobra/src/pntos/cobra/tutorial_plugins/TutorialPosVelOrchestrationPlugin.py
index 99711bd3..3f818ed1 100644
--- a/pntos-cobra/src/pntos/cobra/tutorial_plugins/TutorialPosOrchestrationPlugin.py
+++ b/pntos-cobra/src/pntos/cobra/tutorial_plugins/TutorialPosVelOrchestrationPlugin.py
@@ -31,7 +31,7 @@ from pntos.cobra.config import (
 from pntos.cobra.utils import apply_error_states
 
 
-class TutorialPosOrchestrationPlugin(OrchestrationPlugin):
+class TutorialPosVelOrchestrationPlugin(OrchestrationPlugin):
     fusion_plugin: FusionPlugin
     fusion_strategy_plugin: FusionStrategyPlugin
     inertial_plugin: InertialPlugin
@@ -92,6 +92,7 @@ class TutorialPosOrchestrationPlugin(OrchestrationPlugin):
         # Associate incoming channels with measurement processor labels
         self.measurement_channels = {
             orch_config.position_channel: 'pos',
+            orch_config.velocity_channel: 'vel',
         }
 
         inertial_config = config_from_registry(
@@ -248,6 +249,17 @@ class TutorialPosOrchestrationPlugin(OrchestrationPlugin):
 
         fusion_engine.add_measurement_processor(processor=processor)
 
+        # Create velocity measurement processor and add to fusion engine
+        vel_processor_index = provider.processor_identifiers.index('pinson_velocity')
+        vel_processor = provider.new_processor(
+            processor_index=vel_processor_index,
+            engine=fusion_engine,
+            label='vel',
+            state_block_labels=['pinson15'],
+            config_group='config/gp3d_state_modeling',
+        )
+        fusion_engine.add_measurement_processor(processor=vel_processor)
+
         self.fusion_engine = fusion_engine
 
     def _initialize_inertial_and_fusion_engine(self) -> None:
