diff --git a/pntos-cobra/src/pntos/cobra/tutorial_plugins/TutorialGpsOrchestrationPlugin.py b/pntos-cobra/src/pntos/cobra/tutorial_plugins/TutorialGpsVelOrchestrationPlugin.py
index 4ee8dc07..c5db52c0 100644
--- a/pntos-cobra/src/pntos/cobra/tutorial_plugins/TutorialGpsOrchestrationPlugin.py
+++ b/pntos-cobra/src/pntos/cobra/tutorial_plugins/TutorialGpsVelOrchestrationPlugin.py
@@ -66,6 +66,10 @@ GPS_MEASUREMENT_PROCESSOR_ID = 'pinson_with_ned_fogm_position'
 GPS_MEASUREMENT_PROCESSOR_LABEL = 'gps'
 GPS_MEASUREMENT_PROCESSOR_CONFIG_GROUP = 'config/gp3d_state_modeling'
 GPS_MP_STATE_BLOCK_LABELS = [STATE_BLOCK_LABEL, FOGM_STATE_BLOCK_LABEL]
+VEL_MEASUREMENT_PROCESSOR_ID = 'pinson_velocity'
+VEL_MEASUREMENT_PROCESSOR_LABEL = 'vel'
+VEL_MEASUREMENT_PROCESSOR_CONFIG_GROUP = 'config/gp3d_state_modeling'
+VEL_MP_STATE_BLOCK_LABELS = [STATE_BLOCK_LABEL]
 
 # Inertial parameters
 INERTIAL_GROUP = 'config/inertial'
@@ -76,10 +80,10 @@ PREPROCESSOR_IDS = ['imu_rotator', 'time_adjuster']
 PREPROCESSOR_GROUPS = ['config/imu_rotator', 'config/time_adjuster']
 
 
-class TutorialGpsOrchestrationPlugin(OrchestrationPlugin):
+class TutorialGpsVelOrchestrationPlugin(OrchestrationPlugin):
     """
-    A simple orchestration plugin that incorporates a position measurement processor and two state blocks.
-    It uses a Pinson15 state block as a model for state estimation and a FOGM state block for position bias states.
+    A simple orchestration plugin that incorporates both a position and velocity measurement processor.
+    It uses a Pinson15 state block as a model for state estimation and a FOGM state block for bias states.
     """
 
     mediator: Mediator
@@ -136,9 +140,9 @@ class TutorialGpsOrchestrationPlugin(OrchestrationPlugin):
 
     def _log(self, level: LoggingLevel, message: str) -> None:
         """
-        Send an informational log message to pntOS.
+        Send informational log messages to pntOS.
 
-        If no mediator is provided, manually print the message. Otherwise, pass
+        If no mediator is provided, manually print the messages. Otherwise, pass
         through to the mediator's ``log_message`` method.
         """
         if self.mediator is not None:
@@ -175,6 +179,7 @@ class TutorialGpsOrchestrationPlugin(OrchestrationPlugin):
 
         self.measurement_channels = {
             orch_config.gps_channel: GPS_MEASUREMENT_PROCESSOR_LABEL,
+            orch_config.velocity_channel: VEL_MEASUREMENT_PROCESSOR_LABEL,
         }
         self.alignment_channels = [
             orch_config.gps_channel,
@@ -233,8 +238,8 @@ class TutorialGpsOrchestrationPlugin(OrchestrationPlugin):
         """
         Utility function to assemble the components of and create a fusion engine.
 
-        This function specifically creates a Pinson15 state block, a FOGM state block, and a position measurement processor.
-        If either of these components cannot created, an error will be logged and pntOS will shutdown.
+        This function specifically creates a Pinson15 state block, FOGM state block, position measurement processor, and a velocity measurement processor.
+        If any of these components cannot created, an error will be logged and pntOS will shutdown.
         """
         # Make a fusion engine
         fusion_engine: StandardFusionEngine | None = (
@@ -256,6 +261,7 @@ class TutorialGpsOrchestrationPlugin(OrchestrationPlugin):
         pinson_block = None
         fogm_block = None
         gps_processor = None
+        vel_processor = None
         for plugin in self.state_modeling_plugins:
             provider: StandardStateModelProvider | None = (
                 plugin.new_state_model_provider(StandardStateModelProvider)
@@ -289,6 +295,14 @@ class TutorialGpsOrchestrationPlugin(OrchestrationPlugin):
                     GPS_MP_STATE_BLOCK_LABELS,
                     GPS_MEASUREMENT_PROCESSOR_CONFIG_GROUP,
                 )
+            if VEL_MEASUREMENT_PROCESSOR_ID in provider.processor_identifiers:
+                vel_processor = provider.new_processor(
+                    provider.processor_identifiers.index(VEL_MEASUREMENT_PROCESSOR_ID),
+                    fusion_engine,
+                    VEL_MEASUREMENT_PROCESSOR_LABEL,
+                    VEL_MP_STATE_BLOCK_LABELS,
+                    VEL_MEASUREMENT_PROCESSOR_CONFIG_GROUP,
+                )
 
         # Make state blocks
         if pinson_block is None:
@@ -328,6 +342,16 @@ class TutorialGpsOrchestrationPlugin(OrchestrationPlugin):
             return
         fusion_engine.add_measurement_processor(gps_processor)
 
+        if vel_processor is None:
+            assert provider is not None
+            self._log(
+                LoggingLevel.ERROR,
+                f'Unable to find measurement processor "{VEL_MEASUREMENT_PROCESSOR_ID}" -'
+                + f' cannot initialize filter. Available measurement processors: {provider.processor_identifiers}',
+            )
+            return
+        fusion_engine.add_measurement_processor(vel_processor)
+
         self.fusion_engine = fusion_engine
 
     def process_pntos_message(self, message: Message, sequenced: bool) -> None:
